torch/csrc/api/include/torch/nn/options/activation.h

```diff
@@ -56,5 +57,19 @@ struct LeakyReLUOptions {
   TORCH_ARG(bool, inplace) = false;
 };
 
+// ============================================================================
+
+/// Options for PReLU functional and module.
+struct PReLUOptions {
+  PReLUOptions() {}
```

Comment:
We probably don't need to declare the default constructor as it should be provided automatically.